site/getExternalResources.js

```diff
@@ -0,0 +1,19 @@
+const glob = require('glob');
+const fs = require('fs');
+const { uniq } = require('lodash');
+
+function getExternalResources() {
+  let resources = [];
+  const files = glob.sync('{./{components,docs}/**/*.md,./site/**/*.{less,js,jsx}}');
+  files.forEach((file) => {
+    const content = fs.readFileSync(file, 'utf-8');
+    const pattern = new RegExp('(https://.+.alipayobjects.com/.+.(png|jpg|svg)|)', 'mg');
```

Comment:
should it be `new RegExp('(https://.+\.alipayobjects.com/.+\.(png|jpg|svg))', 'mg');`